docs/DeveloperGuide.md

```diff
@@ -287,15 +287,25 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                 | So that I can…​                                                        |
-|----------|--------------------------------------------|------------------------------|------------------------------------------------------------------------|
-| `* * *`  | new user                                   | see usage instructions       | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person             |                                                                        |
-| `* * *`  | user                                       | delete a person              | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name        | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
-
+| Priority | As a …​           | I want to …​                                                                          | So that I can…​                                                        |
+|----------|-------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------|
+| `* * *`  | new user          | have a quick tutorial of the CLI commands                                             | refer to instructions when I forget how to use the App                 |
+| `* * *`  | receptionist      | quickly retrieve the contact details of clients using simple CLI commands             | contact them promptly about updates or issues                          |
+| `* * *`  | mechanic          | be able to add new cars to the list                                                   | accommodate new clients                                                |
+| `* * *`  | mechanic          | have access to the full history log of each car that includes all visits and services | quickly check on the logs of cars                                      |
```

Comment:
I think this should be * instead of *** as we are unlikely to be able to implement a history log given time constraints.